### PR TITLE
Benefits Minor release note changes

### PR DESCRIPTION
**Pull request by bastosmichael**
### Description

Minor last minute changes to Benefits Intake

### CODE REVIEW

File: src/content/releaseNotes/benefits.mdx [hunk 1]
```diff
@@ -80,6 +80,9 @@
 ---
 
 ## July 23, 2018
+(v1) Announced production support for external use of v1, allowing:
```
Comment (AubreySuter) [inline src/content/releaseNotes/benefits.mdx]: Looks like these got added to the July 23 *2018* list - they should be at the top, for July 23, 2019

Comment (AubreySuter) [reply src/content/releaseNotes/benefits.mdx]: Also, looks like it's under Intake, but should be Claims

Comment (bastosmichael) [reply src/content/releaseNotes/benefits.mdx]: Oops sorry about that, should be fixed and pushed up.

File: src/content/releaseNotes/benefits.mdx [hunk 2]
```diff
@@ -148,29 +148,36 @@ Add appeals service API [#1961](https://github.com/department-of-veterans-affair
 <br />
 <div id="claims" />
 
-# Benefits Claims
-### June 25, 2019
+# Claims
```
Comment (AubreySuter) [inline src/content/releaseNotes/benefits.mdx]: I think we should leave it as "Benefits Claims" but if you think we should switch to just "Claims" I'd be interested to hear your thoughts.

Comment (bastosmichael) [reply src/content/releaseNotes/benefits.mdx]: Fixed as well

**Pull request merged**